src/plugins/CoreTests/src/AnnotationTableObjectTest.h

```diff
@@ -225,6 +225,17 @@ class GTest_CheckAnnotationsNamesInTwoObjects : public XmlTest {
     QString secondDocContextName;
 };
 
+// Compares all annotation tables in two documents. The order of tables is taken into account, the order of annotations
```

Comment:
nit: 1 object == 1 annotation table. It is better to say "Compares annotations in two objects"

Reply:
There can be several annotation tables in one document. By analogy with other checkers, I called "check in two objects", but it is correct to say "check in two documents"